### ISSUE TITLE
Issue loading font assets in Gallery component

### ISSUE THREAD

**Issue by baruchvlz** [mcmakler/components#19]
**What?** On build of project that uses `mcmakler-components-react` as a dependency will throw a build error when importing font assets for the Gallery component.

**Error stack**
```shell
ERROR in ./node_modules/mcmakler-components-react/lib/components/Gallery/assets/fontello.woff2?45416751
    Module build failed: Validation Error
    
    URL Loader Invalid Options
    
    options.limit should be number
    
     @ ./node_modules/css-loader!./node_modules/sass-loader/lib/loader.js?{"sourceMap":false}!./node_modules/mcmakler-components-react/lib/components/Gallery/Gallery.scss 6:10765-10808
```
**Possible Solution**
https://github.com/webpack-contrib/sass-loader/issues/411#issuecomment-296491200

**Comment by Lelith** [comment]
imho the font(s) should come over the design-guide and not be imported in the component style itself. the current setup of the design guide expects the consumer application to include the needed fonts in their layout template.

**Comment by saibs** [comment]
this font is just icon extracted to work with gallery component(not using the entire font for icons and this is different from font-awesome), hence its used with url-loader of webpack.

**Comment by Lelith** [comment]
but when you want to have the font you could use the react-font module which we already use in the select component

**Comment by saibs** [comment]
@abpolat I see the library is using ionicons font, Is it possible for you to replace with any of the above icons, @Lelith can help you with changing necessary styles for loading alternate icons.

**Comment by saibs** [comment]
So, we are going with base64 of font(only the icons), which solves the problem of not loading it externally

**Comment by baruchvlz** [comment]
Fixed with #16
